pyproject.toml

```diff
@@ -46,6 +46,7 @@ dependencies = [
   "pytest-mock",
   "respx",
   "ruff",
+  "yaspin",
```

Comment:
The dependency should be in the `cli` group and replace `halo` (line 116).

Reply:
got it, should have searched for halo in the pyproject before pushing. Thank you!

Reply:
before merging this, attempt the install. This isn't installing yaspin on my machine at the moment